admin/class-wpfep-system-status.php

```diff
@@ -975,12 +975,13 @@ public static function wpfep_get_theme_info()
             $scan_files         = self::scan_template_files(WPFEP_PATH . 'views/');
 
             foreach ($scan_files as $file) {
-                if (file_exists(get_stylesheet_directory() . '/' . $file)) {
-                    $theme_file = get_stylesheet_directory() . '/' . $file;
-                } elseif (file_exists(get_stylesheet_directory() . '/wpfep/' . $file)) {
+                // if (file_exists(get_stylesheet_directory() . '/' . $file)) {
```

Comment:
@frifrafry I'm not sure why you are changing template path in theme folder. Is there any particular reason for that?

It breaks plugin for other users. So please revert back and if you like you can add conditions there.

Reply:
The problem is as follows:
In `admin/class-wpfep-system-status.php` [line 978ff] you search and display template files $file in following locations:
<child-theme>/$file
<child-theme>/wpfe/$file
<parent-theme>/$file
<parent-theme>/wpfe/$file
<plugin>/views/$file

But when you're loading the template in `inc/class-wp-frontend-profile.php` [line 72ff] your using not all these location but only:
<child-theme>/wpfe/$file
<parent-theme>/wpfe/$file
<plugin>/views/$file

So this results in other templates beeing displayed under system status that are really used!

Two possibilities: Either change in `admin/class-wpfep-system-status.php` [line 978ff] or in `inc/class-wp-frontend-profile.php` [line 72ff]

Reply:
Change in `inc/class-wp-frontend-profile.php` would be better to handle this.

Reply:
Okay, I'll change it accordingly.